_posts/2025-11-07-introducing-kubeflow-sdk.md

```diff
@@ -0,0 +1,374 @@
+---
+toc: true
+layout: post
+comments: true
+title: "Introducing the Kubeflow SDK: A Python API for Scalable AI Workloads"
+hide: false
+categories: [sdk, trainer, optimizer]
+permalink: /sdk/intro/
+author: "Kubeflow SDK Team"
+---
+
+# Unified SDK Concept
+
+Scaling AI workloads shouldn't require deep expertise in distributed systems and container orchestration. Whether you are prototyping on local hardware or deploying to a production Kubernetes cluster, you need a unified API that abstracts infrastructure complexity while preserving flexibility. That's exactly what the Kubeflow Python SDK delivers.
+
+As a data scientist, you've probably experienced this frustrating journey: you start by prototyping locally, writing your training script on your laptop. When you need more compute power, it's time to rewrite everything for distributed training. You containerize your code and rebuild images for every change. You write Kubernetes YAMLs, struggle with kubectl and switch between different SDKs - one for training, another for hyperparameter tuning, another for pipelines. Each step in this process requires different tools, different APIs, and different mental models. What if there was a better way?
+
+The Kubeflow community started the Kubeflow SDK & ML Experience Working Group (WG) in order to address these challenges. You can find more information about this WG and the [Kubeflow Community here](https://www.kubeflow.org/docs/about/community/).
```

Comment:
Maybe it would be nice to link our YouTube playlist instead of community page, so readers can directly get up to speed: https://youtu.be/VkbVVk2OGUI?list=PLmzRWLV1CK_wSO2IMPnzChxESmaoXNfrY